app/lib/features/authentication/register/presentation/pages/register_page.dart

```diff
@@ -0,0 +1,227 @@
+          itemAsString: (BebrasBiro bebras) => bebras.userAsString(),
+          compareFn: (item, item2) => item == item2,
+          // dropdownBuilder: _customDropDownExampleMultiSelection,
+            // print(item);
+            // print(item2);
+            // item.i(item2),
+          popupProps: PopupProps.menu(
+            showSearchBox: true,
+            showSelectedItems: true,
+            // disabledItemFn: (String s) => s.startsWith('I'),
+          ),
+          items: dropdownItems,
+          dropdownDecoratorProps: DropDownDecoratorProps(
+            dropdownSearchDecoration: InputDecoration(
+              labelText: labelText,
+              hintText: labelText,
+            ),
+          ),
+          onChanged: print,
+          selectedItem: null,
+        ),
+        // DropdownButtonFormField(
+        //   style: TextStyle(color: Colors.orange),
+        //   decoration: InputDecoration(
+        //     labelText: labelText,
+        //     enabledBorder: OutlineInputBorder(
+        //       borderSide: BorderSide(color: Colors.blue, width: 2),
+        //       borderRadius: BorderRadius.circular(20),
+        //     ),
+        //     border: OutlineInputBorder(
+        //       borderSide: BorderSide(color: Colors.blue, width: 2),
+        //       borderRadius: BorderRadius.circular(20),
+        //     ),
+        //     // filled: true,
+        //     // fillColor: Colors.blueAccent,
+        //   ),
+        //   validator: (value) => value == null ? "Select a country" : null,
+        //   dropdownColor: Colors.blueAccent,
+        //   value: selectedValue,
+        //   onChanged: (String? newValue) {
+        //     print(newValue);
+        //     setState(() {
+        //       selectedValue = newValue!;
+        //     });
+        //   },
+        //   items: dropdownItems,
+        // ),
+      ),
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final size = MediaQuery.of(context).size;
+    return Scaffold(
+      resizeToAvoidBottomInset: false,
+      body: Padding(
+        padding: const EdgeInsets.only(left: 16.0, top: 30.0, right: 16.0),
+        child: Form(
+          key: _formKey,
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.stretch,
+            children: [
+              Image.asset(
+                Assets.bebrasPandaiText,
+              ),
+              buildTextField('Field Name'),
+              buildTextField('Email'),
+              buildTextField('Nama'),
+              buildTextFieldDatePicker('TTL'),
+              buildTextField('Sekolah'),
+              buildTextField('Provinsi'),
+              // buildTextField('Bebas Biro'),
+              buildDropdown('Bebras Biro'),
+              SizedBox(height: 20.0),
+              ElevatedButton(
+                onPressed: () {
+                  // Handle form submission
+                },
+                child: Text('Submit'),
```

Comment:
I can't proceed after click the `Submit` button ^^a
Please continue this first then

Reply:
Also found a small bug that sometimes there is double input in `Bebras Biro` field
<img width="351" alt="image" src="https://github.com/ia-toki/bebras-pandai/assets/1970083/18f4b251-0d8b-486c-9b99-91c97792bb54">